morevna-ep4.msrt

```diff
@@ -66,17 +66,17 @@
 14
 00:01:57,960 --> 00:01:59,580
 [rus] Конечно же, мне совсем неинтересно...
-[eng] Oh yeah, I'm absolutely not interested...
+[eng] Oh yeah, I'm absolutely not interested in who,
 
 15
 00:01:59,590 --> 00:02:02,440
 [rus] Кто это там посреди ночи в подвале у Марьи Моревны за железным замком...
-[eng] Who in the middle of the night in Marya Morevna's cellar  behind the iron lock...
+[eng] In the middle of the night, in Marya Morevna's cellar, behind an iron lock,
 
 16
 00:02:02,580 --> 00:02:06,600
 [rus] Пытается завести движок четвертого поколения с гидравлическим усилителем и пневотурбиной.
-[eng] Is trying to start up the fourth generation engine with hydraulic booster and air turbine.
+[eng] Is trying to start up a fourth generation engine with a hydraulic booster and air turbine.
```

Comment:
I know nothing about the mechanics of giant evil mecha robots, so I'm just assuming he has one hydraulic booster and one air turbine. If he has more, then they should be made plural as necessary.